lib/core/interfaces/IOperationRateLimiter.ts

```diff
@@ -0,0 +1,19 @@
+import TransactionModel from '../../common/models/TransactionModel';
+
+/**
+ * Rate limits operations. only allow n number of highest fee transactions to pass.
+ */
+export default interface IOperationRateLimiter {
+
+  /**
+   * Given an array of transactions, return a subset of them which have high enough fee within their block to be processed. In the returned object,
+   * also contains transactions that belong to an incomplete block at the end of the array
```

Comment:
Sanity: seems that this part of the comment is not true? Doesn't look like that's what's been implemented.